machines/slurm.train-neural-lam.sh

```diff
@@ -1,15 +1,6 @@
 #!/bin/bash -l
-#SBATCH --job-name=HAS-NeuralLam
-#SBATCH --time=1-00:00:00
-#SBATCH --nodes=1
```

Comment:
these should also be removed in `machines/slurm.mllam-data-prep.sh` I think, right? As in these values should come from `params.yaml` now